makefile

```diff
@@ -98,7 +98,8 @@ $(if $(wildcard $(THIS_MK)),,$(error Makefile not found "$(THIS_MK)"))
 MFEM_DIR ?= $(patsubst %/,%,$(dir $(THIS_MK)))
 MFEM_REAL_DIR := $(realpath $(MFEM_DIR))
 $(if $(MFEM_REAL_DIR),,$(error Source directory "$(MFEM_DIR)" is not valid))
-SRC := $(if $(MFEM_REAL_DIR:$(CURDIR)=),$(MFEM_DIR)/,)
+override MFEM_DIR := $(MFEM_REAL_DIR)
+SRC := $(if $(MFEM_DIR:$(CURDIR)=),$(MFEM_DIR)/,)
```

Comment:
I want to keep `MFEM_DIR` and `MFEM_REAL_DIR` as they are - this change is not really needed here. Or is it?

Reply:
The last commit should revert the `MFEM_DIR` changes